index.css

```diff
@@ -0,0 +1,318 @@
+body{
+    margin: 0;
+    padding: 0;
+    width: 100vw;
+    overflow-x: hidden;
+    /* position: relative; */
+    
+}
+#header{
+    margin: 0;
+    /* position: sticky; */
+    padding: 10px 20px;
+    /* background-color: aqua; */
+    display: flex;
+    width: 100vw;
+    /* overflow-x: hidden; */
+    /* align-items: center;
+    justify-content: center; */
+}
```

Comment:
consider using two-space indentation as opposed to tab indentation. It is an agreed upon best practice.

Reply:
- [ ] You may also consider telling the author to delete the commented lines of code for it's not a best practice to have them in the code base